homeassistant/components/group/__init__.py

```diff
@@ -31,6 +31,12 @@
 from homeassistant.helpers.typing import ConfigType
 from homeassistant.loader import bind_hass
 
+# Ensure group config_flow is imported so it does not need the import
+# executor since config_flows are preloaded when the component is loaded.
+# Even though group is pre-imported above we would have still had to wait
```

Comment:
The sentence isn't completely accurate after the move.

Reply:
I remember reading that over again and it seemed OK at the time, but probably some confirmation bias going on there. I'll rewrite it after walking the dog and I'm a little further away from it.

Reply:
https://github.com/home-assistant/core/pull/113702 hopefully that makes more sense